### DISCUSSION TITLE
Menu Principal

### DISCUSSION THREAD

**Discussion by roverem** [gamerevar/proyecto-comunitario-1-gd#71] [Game Design]
Para el Menu principal se me ocurre que podríamos obviar los menúes clásicos con botones con texto (ref 1)

En vez de eso usar el contexto en que se encuentran los personajes para presentarle al jugador la selección de las opciones.

Los personajes que protagonizan el juego se encuentran viajando en un auto entre misiones, por lo tanto tiene sentido que para elegir comenzar una misión o cambiar un setting el jugador tenga que interactuar con alguno de los controles del vehiculo en cuestión.

Entonces el jugador vería algo como lo siguiente al ingresar al menu principal del juego:

![image](https://user-images.githubusercontent.com/7843438/103328972-9480ea80-4a39-11eb-9873-4c5ab9a2f360.png)

Donde realmente lo que se le presentaría al usuario sería algo como lo siguiente:

![image](https://user-images.githubusercontent.com/7843438/103329506-abc0d780-4a3b-11eb-893f-d150c451534c.png)

Es decir que el jugador puede hacer mouse-over en el volante y el mismo reacciona (con un glow, una animación, etc a definir).

Y al clickearlo el usuario selecciona la opción en cuestión.

(Nunca está de mas aclarar: el usuario no vería los textos de las opciones para no romper la inmersión)

Por el momento las opciones serían:

- Nueva Partida

- Settings (Gráficos low/high)

- Salir

No veo la necesidad de agregar mas. (Quizás fuera necesario un botón para borrar progreso.)

Mas abajo algunas referencias. Ref 2 y ref 3 son para ver otras posibles botoneras de vehículos militares que puede servir de inspiración.

ref 1
![image](https://user-images.githubusercontent.com/7843438/103329144-36a0d280-4a3a-11eb-922a-494b0b2466ba.png)

ref 2
![image](https://user-images.githubusercontent.com/7843438/103329090-f6415480-4a39-11eb-94df-bde5d52dedad.png)

ref 3
![image](https://user-images.githubusercontent.com/7843438/103329099-06f1ca80-4a3a-11eb-9876-0e13d820a2d1.png)

**Comment by gamerevar** [comment]
Se creo el issue [#81](https://github.com/gamerevar/proyecto-comunitario-1-gd/issues/81). Continuarlo ahi.
